### PR TITLE
Addition of PhDIntern Page

### PR DESCRIPTION
**Pull request by JRPearson500**
# Description

❓**What**: Relating to Issue #23 - Add PhD Internship content

🧠**Why?**: To connect this site to the intern microsite

👨‍💻**How?**: Addition of new tab between Data Science in NHS England and Articles

# Checklist:
Have checked for the following:
- [X] The website still builds correctly, and you can view it using `mkdocs serve`.
- [X] There are no new "warnings" from mkdocs
- [X] Does your page follow the [page template](https://nhsdigital.github.io/rap-community-of-practice/example_RAP_CoP_page/) (or [here in Markdown](https://github.com/NHSDigital/rap-community-of-practice/blob/main/docs/example_RAP_CoP_page.md))? (**need to make a new one specific to NHSE Data Science**)
- [X] Spelling errors
- [X] Consistent capitalization
- [X] Consistent numbers
- [X] Material features incorrectly implemented: search for code blocks and markers (e.g. !!!).
- [X] Code snippets don't work
- [X] Images not working
- [X] Links not working

## Where it was tested

- MacOS Sonoma 14.3
- devcontainer.json describes further settings

### CODE REVIEW

File: docs/PhDInterns.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Available outputs from previous projects can also be seen at [Previous Projects](https://nhsx.github.io/nhsx-internship-projects/completed.html) on the microsite
+
+### Current Projects 
+Currently our interns are working on the following projects in two waves.  These are the original briefs they applied to and their work and outputs will be available on our organisation GitHub.
+
+
+
+|**Wave 6** | February - July 2024 |
+|---|---|
+|![area](https://img.shields.io/badge/NLP-yellow) &nbsp; ![technique](https://img.shields.io/badge/Curation-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/MultiModal-blueviolet)|[NHS Language Corpus Extension](https://nhsx.github.io/nhsx-internship-projects/nhs-language-corpus/)|
```
Comment (amaiaita) [inline docs/PhDInterns.md:25]: do we want to use the tags feature for this instead? join them up with the projects page?

File: docs/PhDInterns.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+|![area](https://img.shields.io/badge/Explainability-orange) &nbsp; ![technique](https://img.shields.io/badge/Fairness-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/MultiModal-blueviolet) | [Understanding Fairness and Explainability in Multi-modal Approaches within Healthcare](https://nhsx.github.io/nhsx-internship-projects/advances-modalities-explainability/)|
```
Comment (amaiaita) [inline docs/PhDInterns.md:26]: also, we dont have these projects in the projects page, is it worth transferring them over, with some detail on progress and linking to that?

File: docs/PhDInterns.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+hide:
+  - navigation
+---
+
+# PhD Intern Scheme
```
Comment (amaiaita) [inline docs/PhDInterns.md:6]: should it not be either "PhD Internship Scheme" or "PhD Interns Scheme"?

**Review by amaiaita — COMMENTED** [review]
some thoughts on integrating this better with the rest of the website, but looks good otherwise

File: mkdocs.yml [hunk 4]
```diff
@@ -19,6 +19,7 @@ nav:
   - What is Data Science?: 
     - ... | flat | what_is_data_science/*.md
   - About the team: about.md
+  - PHDInterns: PhDInterns.md
```
Comment (amaiaita) [inline mkdocs.yml:22]: i think this should come after the articles and projects sections in the structure, but let me know if you dont agree

Comment (jrpearson-nhs) [reply mkdocs.yml:22]: Will Update

File: docs/PhDInterns.md (hunk 3 above)
Comment (jrpearson-nhs) [reply docs/PhDInterns.md:6]: Will correct to be consistent to "PhD Internship Scheme"

File: docs/PhDInterns.md (hunk 1 above)
Comment (jrpearson-nhs) [reply docs/PhDInterns.md:25]: I'll have a go at updating to this and feedback

File: docs/PhDInterns.md (hunk 2 above)
Comment (jrpearson-nhs) [reply docs/PhDInterns.md:26]: Personally would prefer to keep them separate for clarity.

**Review by amaiaita — APPROVED** [review]

**Pull request merged**